db/migrate/20210912173535_add_index_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToTags < ActiveRecord::Migration[6.1]
```

Comment:
no need to create seperate files for now can be added in create migration itself.

Reply:
Yeah. I removed this file in https://github.com/lokendrasingh-1mg/StackOverflowRailsClone/pull/1/commits/7d195e86cf29e820339a8e8e222550b5b4d7148a